src/components/general/TextInput/TextInput.stories.tsx

```diff
@@ -45,7 +45,7 @@ const TextInputStory = () => {
     );
 
     return (
-        <div style={{ display: 'flex', padding: '8px', flexDirection: 'column' }}>
+        <div style={{ display: 'flex', padding: '8px', flexDirection: 'column', maxWidth: '328px' }}>
```

Comment:
Since we're removing the max-width of the text input, should we perhaps showcase that it _can_ grow to extreme widths?

Reply:
Sure! I saw that the textarea was set up the same way, so it makes sense that they match. I've removed the max with from both the text input and the searchable dropdown stories.